README.md

```diff
@@ -44,18 +48,54 @@ prototype(Vendor.Site:Component) < prototype(PackageFactory.AtomicFusion:Compone
             }
 
             description = Neos.Fusion:Tag {
-                tagName = 'p'
                 content = ${props.description}
             }
         }
     }
 }
 ```
 
+### 2. Content Mapping
+
+```
+#
+# Map node content zo a presentational component 
+# 
+# instead of Neos.Neos:Content PackageFactory.AtomicFusion:Content 
+# is used base prototype 
+#
+prototype(Vendor.Site:ExampleContent) < prototype(PackageFactory.AtomicFusion:Content) {
+	renderer = Vendor.Site:Component {
+	
+		# 
+		# pass boolean property 'bold' from the
+		# node to the component
+		#
+		bold = ${q(node).property('bold')}	
+	
+		#
+		# use the editable component to pass an editable 
+		# inline-tag for the property 'title'
+		#
+		title = PackageFactory.AtomicFusion:Editable {
+			property = 'title'
+			inline = true
```

Comment:
Not too sure about `inline` anymore. This might be confusing, since we are mapping **inline editables** at this point... Maybe `block = false` is less ambiguous